### PR TITLE
check whether integer is even or odd using bit manupulation

### PR DESCRIPTION
**Pull request by luciferx48**
### Describe your change:

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [ ] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Review by CaedenPH — APPROVED** [review]
Looks good

**Comment by CaedenPH** [comment]
@luciferx48 
I forgot to mention, could you add the algorithm to the DIRECTORY.md file please

**Comment by luciferx48** [comment]
@CaedenPH thnx for reviewing, Directory.md updated

**Review by CaedenPH — CHANGES_REQUESTED** [review]
In the DIRECTORY.md, algorithms should be in alphabetical order

**Review by CaedenPH — APPROVED** [review]
@cclauss Can we get your review?

File: bit_manipulation/even_or_not.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    True
+    >>> is_even(100)
+    True
+    >>> is_even(101)
+    False
+    """
+    if number & 1 == 0:
+        return True
+    else:
+        return False
```
Comment (cclauss) [inline bit_manipulation/even_or_not.py:34]:
```suggestion
    return number & 1 == 0
```

**Comment by luciferx48** [comment]
@cclauss can you merge or close this PR

**Review by cclauss — APPROVED** [review]

**Pull request merged**